spec/requests/users_controller_spec.rb

```diff
@@ -5186,6 +5186,90 @@ def revoke(user, skip_remote: false)
     end
   end
 
+  describe "#reset_recent_searches" do
+    fab!(:user) { Fabricate(:user) }
+
+    it 'does nothing for anon' do
```

Comment:
```suggestion
    it 'returns the right response for anon' do
```

Minor but I feel like the controller action is actually still doing something.